.pre-commit-config.yaml

```diff
@@ -66,28 +66,18 @@ repos:
         ]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.4.1
+    rev: v1.7.1
     hooks:
       - id: mypy
 
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.4.1
-    hooks:
-      - id: mypy
-        args: [
-          --disallow-untyped-defs,
```

Comment:
Hi @dstansby. The purpose of this admittedly duplicated block was to be *harsher* on non-test code, i.e. require typed-defs everywhere.